source/tutorials/nixos/installing-nixos-on-a-raspberry-pi.md

```diff
@@ -191,10 +191,10 @@ $ sudo -i
 
 - Once you have a working OS, try upgrading it with `nixos-rebuild switch --upgrade` to install more recent package versions, and reboot to the old configuration if something broke.
 - To enable hardware acceleration for a nice graphical desktop experience, you can enable the [`nixos-hardware` module](https://github.com/nixos/nixos-hardware) by adding the following to your configuration:
-
-```nix
-imports = [
-  "${builtins.fetchGit { url = "https://github.com/NixOS/nixos-hardware.git"; }}/raspberry-pi/4"
-];
-```
+  ```nix
+  imports = [
+    "${builtins.fetchGit { url = "https://github.com/NixOS/nixos-hardware.git"; }}/raspberry-pi/4"
+  ];
+  ```
+  Ensure that you have `git` available in your current shell first ([via](https://github.com/NixOS/nix/issues/1923#issuecomment-1665509352)).
```

Comment:
This should be highlighted as a warning, with concrete instructions how to do that, e.g. `nix-shell -p` and a NixOS box for the configuration option. We can link to the issue (which will hopefully be closed eventually with the current libgit rewrite going on), but only as a side note (please use descriptive a link text that can stand on its own).

Reply:
Try `nix-shell --run devmode` for a preview (see `README.md`).